### DISCUSSION TITLE
Why install as dev?

### DISCUSSION THREAD

**Discussion by markedwards** [caching-tools/next-shared-cache#572] [General]
Why do the installation docs point to installing as `--dev`?  Isn't this (and Redis) a runtime dependency?

**Comment by better-salmon** [comment]
Hello @markedwards! I wanted point it out that this package must be only on the server side and never bundled to the client. But this approach breaks package semantics. I've edited the docs. Thank you for your question!

**Reply by markedwards** [reply]
Yes, I didn't imagine it being bundled into the client in any case.  But it seems to me that it will commonly be running in the production server, so it should be a normal dependency.  I just wanted to see if I was missing something.

Thanks!
